src/content/reference/manifest.mdx

```diff
@@ -254,6 +254,23 @@ deploy:
 - `virtualService`: the name of the virtual service you want to divert.
 - `hosts`: the list of hosts you want to divert in the developer namespace. Requests to these virtual services will inject the header `x-okteto-divert`.
 
+#### Deploy remotely
+
+Using the `--remote` flag lets you run the `okteto deploy` commands directly in your Okteto cluster. 
+
+When executing `okteto deploy` with this flag, Okteto will automatically synchronize your local folder, and then it will execute the `build` and `deploy` sections of the manifest directly in your Okteto cluster, using the image defined in the manifest. 
+
+With this, you don't have to worry about installing tools like `helm`, `kubectl`, and others on your local machine. Instead, you define a container image with all your tools and you, and everyone else on your team, get the same experience with zero configuration required. 
+
+```yaml
+deploy:
+  image: okteto/pipeline-installer:1.7.6
```

Comment:
this image doesn't exist or is not public. can we update it the sample to it's functional?